BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -355,15 +357,21 @@ class BraintreeClient @VisibleForTesting internal constructor(
     }
 
     private fun sendAnalyticsTimingEvent(endpoint: String, timing: HttpResponseTiming) {
-        val cleanedPath = endpoint.replace(Regex("/merchants/([A-Za-z0-9]+)/client_api"), "")
-        sendAnalyticsEvent(
-            CoreAnalytics.apiRequestLatency,
-            AnalyticsEventParams(
-                startTime = timing.startTime,
-                endTime = timing.endTime,
-                endpoint = cleanedPath
-            )
+        var cleanedPath = endpoint.replace(Regex("/merchants/([A-Za-z0-9]+)/client_api"), "")
+        cleanedPath = cleanedPath.replace(
+            Regex("payment_methods/.*/three_d_secure"), "payment_methods/three_d_secure"
         )
+
+        if (cleanedPath != "/v1/tracking/batch/events") {
```

Comment:
What is the context on this part of the change? What is that path used for that isn't sent to analytics?

Reply:
On iOS it was added to ensure that we don't send analytics events basically saying that we sent analytics since that's the FPTI path ([here](https://github.com/braintree/braintree_ios/blob/7b81129478fb3302b2ffacdbb4ade80e561faef7/Sources/BraintreeCore/BTAPIClient.swift#L406)). Though adding breakpoints on Android it seems like we use a totally different post method just for analytics so this is likely not needed currently for Android.

Reply:
Updated: https://github.com/braintree/braintree_android/pull/1080/commits/a72b21a6a679cb8b204bcbb3ff3c65cff2ae25e8